Include the POSIX headers blas_server.c needs on QNX too.

blas_server.c includes <sys/resource.h>, <signal.h>, <dlfcn.h> and friends
only for a list of named systems; on QNX the thread-creation error path then
fails to compile ("storage size of 'rlim' isn't known").

Upstream-Status: QNX-specific

--- a/driver/others/blas_server.c
+++ b/driver/others/blas_server.c
@@ -70,7 +70,7 @@
 /*********************************************************************/
 
 #include "common.h"
-#if defined(OS_LINUX) || defined(OS_NETBSD) || defined(OS_DARWIN) || defined(OS_ANDROID) || defined(OS_SUNOS) || defined(OS_FREEBSD) || defined(OS_OPENBSD) || defined(OS_DRAGONFLY) || defined(OS_HAIKU)
+#if defined(OS_LINUX) || defined(OS_NETBSD) || defined(OS_DARWIN) || defined(OS_ANDROID) || defined(OS_SUNOS) || defined(OS_FREEBSD) || defined(OS_OPENBSD) || defined(OS_DRAGONFLY) || defined(OS_HAIKU) || defined(__QNX__)
 #include <dlfcn.h>
 #include <errno.h>
 #include <signal.h>
